src/Components/Error/error.css

```diff
@@ -0,0 +1,70 @@
+
+*{
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+}
+
+body{
+	background-color: whitesmoke;
+}
+
+.error-container{
+	display: grid;
+	/*width: 100vw;*/
+	height: 100vh;
+	align-content: center;
+	justify-content: center;
+}
+
+.error-message{
+	display: grid;
+	width: 45vmax;	
+	height: fit-content;
+	/*margin: 10vmax auto;*/
+	border-radius: 2rem;
+	align-content: center;
+	justify-content: center;
+	background-color: lightgray;
+	padding: 3rem;
+	box-shadow: 5px 5px 20px 5px dimgray;
+}
+
+span{
+	font-size: 1rem;
+	color: olivedrab;
+	color: rebeccapurple;
+}
+
+
+.body-1{
+	display: grid;
+	text-align: justify;
+	align-content: space-around;
+}
+
+.header{
+	align-self: start;
+	justify-self: left;
+	width: 100%;
+}
+
+.hr{
+	height: 5px;
+	background-color: black;
+	width: 100%;
+	align-self: start;
+	margin: 4px auto;
+	border-radius: 5px;
+}
+.hr-2{
+	height: 1px;
+	background-color: black;
+	width: 100%;
+	margin: 2px auto;
+	border-radius: 5px;
+}
+
+.body-2{
+		text-align: justify;
```

Comment:
Fix indentation.

Reply:
Yeah I will apply indentation of tab size of 4-spaces to it.